extensions/stackdriver/stackdriver.cc

```diff
@@ -732,13 +780,38 @@ void StackdriverRootContext::evaluateExpressions(
   }
 }
 
+void StackdriverRootContext::evaluateMetricsExpressions(
+    std::unordered_map<std::string,
+                       std::vector<std::pair<std::string, std::string>>>&
+        overrides) {
+  for (const auto& expression : metrics_expressions_) {
+    std::cout << "adding expression: " << expression.tag
+              << ", for metric: " << expression.metric << "\n";
+    std::string value;
+    if (!evaluateExpression(expression.token, &value)) {
+      LOG_WARN(absl::StrCat("Could not evaluate expression: ",
+                            expression.expression));
+      continue;
+    }
+    overrides[expression.metric].emplace_back(
+        std::make_pair(expression.tag, value));
```

Comment:
I don't think you need to make_pair? Should work if you directly `emplace_back(expression.tag, value)`, so that we can avoid a copy.

Reply:
done.